nodedb.go

```diff
@@ -452,7 +452,7 @@ func (ndb *nodeDB) DeleteVersionsFrom(version int64) error {
 	// Next, delete orphans:
 	// - Delete orphan entries *and referred nodes* with fromVersion >= version
 	// - Delete orphan entries with toVersion >= version-1 (since orphans at latest are not orphans)
-	err = ndb.traverseOrphans(func(key, hash []byte) error {
+	err = ndb.traverseRange(orphanKeyFormat.Key(version-1), PrefixEndBytes(orphanKeyFormat.Key()), func(key, hash []byte) error {
```

Comment:
[traversePrefix](https://github.com/cosmos/iavl/blob/66f11ae7fc2126c45a00beae9d5d8f33445d906f/nodedb.go#L817) when call [IteratePrefix](https://github.com/cosmos/cosmos-db/blob/23a8145386c0/prefixdb_iterator.go#L10) do same job   
```suggestion
	err = ndb.traverseOrphansVersion(version-1, func(key, hash []byte) error {
```

Reply:
yeah, you are right, don't remember why I end up with that solution ;D

Reply:
no, we must use `traverseRange`, because we need to traverse from `version - 1` to the end, `traverseOrphansVersion` only traverse single toVersion.